beps/bep_0052.rst

```diff
@@ -444,7 +444,8 @@ to the leaf and piece layers. Index is the offset in hashes
 of the first requested hash in the base layer.
 Index MUST be a multiple of length, this includes zero.
 Length is the number of hashes to include from the base layer.
-Length MUST be equal-to-or-greater-than two and a power of two.
+Length MUST be equal-to-or-greater-than two and a power of two unless it
+gets truncated by the end of the layer.
```

Comment:
This is mimicking

'request' messages contain an index, begin, and length. The last
two are byte offsets. Length is generally a power of two unless it
gets truncated by the end of a file. All current implementations use
2^14 (16 kiB), and close connections which request an amount greater than
that.